gulpfile.js

```diff
@@ -805,7 +805,8 @@ function printNobuildHelp() {
  */
 function printConfigHelp(command) {
   if (!isTravisBuild()) {
-    log(green('Building the runtime for local testing with the'),
+    log(green('Building version'), cyan(internalRuntimeVersion),
+        green('of the runtime for local testing with the'),
```

Comment:
With #22109, this will have to move here:

https://github.com/ampproject/amphtml/blob/3e110a55a416f42dcd879c2779b67014d25e22c8/build-system/tasks/helpers.js#L495-L507

Reply:
Done